src/seedsigner/gui/screens/seed_screens.py

```diff
@@ -438,14 +444,55 @@ def __post_init__(self):
 
 
 
+@dataclass
+class SeedBIP85FinalizeScreen(ButtonListScreen):
+    child_fingerprint: str = None
+    bip85_index: int = None
+    is_bottom_list: bool = True
+    button_data: list = None
+
+    def __post_init__(self):
+        self.title = _("BIP-85 Child Seed")
+        super().__post_init__()
+
+        self.child_fingerprint_icontl = IconTextLine(
+            icon_name=SeedSignerIconConstants.FINGERPRINT,
+            icon_color=GUIConstants.INFO_COLOR,
+            icon_size=GUIConstants.ICON_FONT_SIZE + 12,
```

Comment:
The existing call on line 436 ideally wouldn't have hard-coded a pixel size adjustment value. It's best to avoid replicating it here since we'll eventually need to refactor all the hard-coded pixel values. In general, a relative value (e.g. 1.5x the size of the accompanying font) will be the most flexible.